aio/content/examples/rx-library/src/simple-creation.2.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { docRegionInterval } from './simple-creation.2';
+
+describe('simple-creation.2', () => {
+  beforeEach(() => {
+    jasmine.clock().install();
+  });
+
+  afterEach(() => {
+    jasmine.clock().uninstall();
+  });
+
+  it('should create an Observable that will publish a value on an interval', () => {
+    const console = {log: jasmine.createSpy('log')};
+    const subscription = docRegionInterval(console);
+    jasmine.clock().tick(1001);
```

Comment:
```suggestion
    jasmine.clock().tick(1000);
```

Also, to make sure it works as expected, let's add an assertion that there is nothing emitted after 999ms.